apps/src/applab/designElements/ImagePickerPropertyRow.jsx

```diff
@@ -0,0 +1,45 @@
+var React = require('react');
+
+var PropertyRow = React.createClass({
+  propTypes: {
+    initialValue: React.PropTypes.string.isRequired,
+    handleChange: React.PropTypes.func
+  },
+
+  getInitialState: function () {
+    return {
+      value: this.props.initialValue
+    };
+  },
+
+  handleChangeInternal: function (event) {
+    this.changeImage(event.target.value);
+  },
+
+  handleButtonClick: function () {
+    Applab.onDesignModeManageAssets(this.changeImage, 'image');
```

Comment:
I think ideally this component would `require('../assetManagement/AssetManager.jsx')` but there's a little weirdness where `AssetManger` is just the modal content, and the actual modal wrapper is dependent on `studioApp.showModalDialog`.  I've added a TODO in https://github.com/code-dot-org/code-dot-org/commit/0f16c434b4482aa51e701ed748efd3a6079fad3f#diff-edebf8b430d138b0bc5eedded7296816R21 until we can make the function for showing a dialog its own `require`.